src/files/infra/repositories/transformers/filePreviewTransformers.ts

```diff
@@ -0,0 +1,40 @@
+import { PublicationStatus } from '../../../../core/domain/models/PublicationStatus'
+import { FilePreview } from '../../../domain/models/FilePreview'
+import { FilePreviewPayload } from './FilePreviewPayload'
+
+export const transformFilePreviewPayloadToFilePreview = (
+  filePreviewPayload: FilePreviewPayload
+): FilePreview => {
+  const publicationStatuses: PublicationStatus[] = []
+  filePreviewPayload.publicationStatuses.forEach((element) => {
+    publicationStatuses.push(element as unknown as PublicationStatus)
+  })
+  return {
+    name: filePreviewPayload.name,
+    url: filePreviewPayload.url,
+    imageUrl: filePreviewPayload.image_url,
+    ...(filePreviewPayload.image_url && { imageUrl: filePreviewPayload.image_url }),
```

Comment:
I think here we are setting imageUrl in two places, line 15 and then conditionally in line 16.